shell/plugins/dashboard-store/mutations.js

```diff
@@ -51,6 +51,22 @@ export function load(state, { data, ctx, existing }) {
 
   cache.generation++;
 
+  /**
+   * Resource counts are contained within a single 'count' resource with a 'counts' field that is a map of resource types
+   * When counts are updated through the websocket, only the resources that changed are sent so we can't load the new 'count' resource into the store as we would another resource
+   */
+  if (type === COUNT && cache.haveAll) {
+    const newCounts = data.counts;
+    const countResource = cache.list[0];
+    const existingCounts = countResource.counts;
+    // creating a new object like this and calling Vue.set ensures the rest of the dash picks up these nested count value changes
+    const updatedCounts = { ...existingCounts, ...newCounts };
+
+    Vue.set(countResource, 'counts', updatedCounts);
```

Comment:
I didn't think this would work and needed to be `Vue.set(cache.list[0]` instead... definitely a TIL

Reply:
there was a bit of trial and error here :)